client/src/router/index.js

```diff
@@ -22,6 +17,11 @@ const routes = [
     component: Publication,
     props: true,
   },
+  {
+    path: '*',
+    name: 'List',
+    component: List,
+  },
 ]
 
 const router = new VueRouter({
```

Comment:
Can probably remove the qs dependency now?

Reply:
👍